test/helpers.js

```diff
@@ -0,0 +1,30 @@
+// Helper function to determine whether this browser is supported by 
+// impress.js or not. Copied from impress.js itself.
+var _impressSupported = function() {
```

Comment:
> If you want to suggest a more proper way of encapsulating this function into some helper namespace

The current code is not doing any encapsulation, it is exposed as a global variable with or without underscore. There is no difference between prefixing with underscore or not, in either case we need to take care to acknowledge the globals that are being used in testing and take care for not overriding it.

If one is worried about too many global variables, it is always possible to put the code into a closure and expose a single "helper" namespace that will contain all necessary utility classes as properties of the returned object.

Also, I would argue that this code is so similar to the one in the core that there is no problem in exposing it only for testing purposes to prevent code duplication, what matters is what is documented, not what is exposed (of course, it is necessary a proper public API documentation first).